distribution/ltp/lite/grab_corefiles.sh

```diff
@@ -25,7 +25,7 @@ echo "List of files to pack: $bin_core_list"
 if [ -n "$bin_core_list" ]; then
     tar cfvz binaries_and_corefiles.tar.gz $bin_core_list
     rhts_submit_log -l binaries_and_corefiles.tar.gz
-    report_result unexpected_corefile_found PASS 0
+    report_result unexpected_corefile_found WARN 1
```

Comment:
This was reporting core files caused by OS bugs and hence determined to not be reliable to use in CKI (see commit dd32969f29)